### DISCUSSION TITLE
How do I build florisboard?

### DISCUSSION THREAD

**Discussion by magnus-ISU** [florisboard/florisboard#1800] [Q&A]
I haven't updated to .15 yet because I felt certain things were a regression. However, I still want to be involved in development on this keyboard, as it is the best open-source keyboard I've found. Plus I could try to fix some of the issues I run into.

I opened it up and solved #1664 for myself quickly - however, I can't build florisboard to try it. I'm pretty sure I have android studio properly configured, but gradle build fails with 

```
C/C++: ninja: error: '/home/mag/Documents/code/android/florisboard/app/src/main/icu4c/prebuilt/jniLibs/arm64-v8a/libicuuc.a', needed by '/home/mag/Documents/code/android/florisboard/app/build/intermediates/cxx/RelWithDebInfo/1d1w5y5k/obj/arm64-v8a/libflorisboard-native.so', missing and no known rule to make it
```

what

**Comment by patrickgold** [comment]
You need to initialize the git submodules, which contains files required to build this project: 

``git submodule update --init --recursive``

Then it should work.
